runtime/tutor.txt

```diff
@@ -169,7 +169,10 @@ _________________________________________________________________
 
  * Type i to enter Insert mode and type text. Type <ESC> to
    return to Normal mode.
-
+   * Use a to enter Insert mode after the current selection
+     instead of before.
+   * Use I or A to enter Insert mode at the start or end of
+     a line respectively.
```

Comment:
<kbd>I</kbd> prepends to the line starting after any whitespace. I'm not sure it's worth mentioning that here. I suppose we could split out bullets on <kbd>I</kbd> and <kbd>A</kbd> to be a bit more explicit

Reply:
Sure, that makes sense. Added.